### PR TITLE
Improve Dimensional Sacrifice with Achievement 118 and autobuyer

### PR DESCRIPTION
**Pull request by Donaldino7712**
Removes unused data in the player object (`player.infinity.upgradeBits`), and updates the dimensional sacrifice button so that it no longer flickers with r118 and automatic sacrifice enabled.

### CODE REVIEW

**Comment by Hirame1** [comment]
In this branch, when I import an old save from a savebank (I tried with a first save in the post-break), it automatically imports as a moden ui save, then it shows a modal to swap it to `modern` (then it swap to `classic`).
It should import as a `classc` ui save, then swap to `modern`.
Could you check it?

image

**Comment by Donaldino7712** [comment]
Can confirm that this happens, I have no clue what's causing this.
I'll to try to find the cause and I'll push a fix if I find one.

**Comment by Donaldino7712** [comment]
The issue seems to be related to the fact that `player.infinity.upgradeBits` doesn't exist on the save

**Comment by Donaldino7712** [comment]
I see the issue:
The save that you are trying to import is from the version `12,1`.
The variable was added in version `13`, which explains why it doesn't exist on the old save.
Now, should I modify the `13` migration to add the variable or put the `delete` in the `26` migration in a `try/catch`?
I think that the first solution would be better in this case.

**Comment by Donaldino7712** [comment]
@Hirame1 Could you try now?

**Comment by Hirame1** [comment]
Hmm, it looks like we need to implement a proper fix for this.
The save exported from the Android (export to web/steam) doesn't have `player.infinity.upgradeBits`.
~~or just forget about this matter at all~~

Sample save: https://paste.ee/p/3GyMd

**Comment by Donaldino7712** [comment]
If it doesn't exist, than the save will behave just like the one you tried before my fix.

I'm honestly not sure what we should do to fix this issue.

**Comment by HexaVault** [comment]
> If it doesn't exist, than the save will behave just like the one you tried before my fix.
> 
> I'm honestly not sure what we should do to fix this issue.

Do a try catch, or just
`if (!!player.infinity?.upgradeBits) delete player.infinity.upgradeBits`

**Comment by Donaldino7712** [comment]
I'll push this fix once I can, thanks for the sugestiom Hexa!

**Review by Hirame1 — APPROVED** [review]
It is now working well 👍

File: src/components/tabs/antimatter-dimensions/ModernAntimatterDimensionsTab.vue [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <script>
+import { Achievement } from "../../../core/globals";
```
[the remark this answers is not in the document]
Comment (HexaVault) [reply src/components/tabs/antimatter-dimensions/ModernAntimatterDimensionsTab.vue]: Probably just an oversight when copypasting this from be port which does the same thing. I don't know if its necessary, but i don't think so.

[the remark this answers is not in the document]
Comment (Donaldino7712) [reply src/components/tabs/antimatter-dimensions/ModernAntimatterDimensionsTab.vue]: VSCode added it for me. It isn't actually necessary for the script to function.

**Review by cyip92 — APPROVED** [review]

**Pull request merged**